### PR TITLE
Added warning about password in sample client config file

### LINKED ISSUE #37132: Warn about passwords in kubeconfig
**This is a Feature Request**

**What would you like to be added**
Add a comment to the example Kubernetes client config in https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/

```yaml
users:
- name: developer
  user:
    client-certificate: fake-cert-file
    client-key: fake-key-file
- name: experimenter
  user:
    password: some-password # ADD A WARNING HERE!
    username: exp
```

**Why is this needed**
Storing a password in a Kubernetes client config is risky, and we have an opportunities to call out other options.

**Comments**
A good alternative is a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) and separate secret storage.

/language en
/sig security

### PR DESCRIPTION
**Pull request by sreeram-venkitesh**
Closes #37132

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 1]
```diff
@@ -153,6 +153,10 @@ users:
     client-key: fake-key-file
 - name: experimenter
   user:
+    # Storing password in Kubernetes client config is risky.
+    # A better alternative would be to use a credential plugin
+    # and storing the credentials on a separate secret.
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:158]: (nit)
```suggestion
    # and store the credentials separately.
``

File: content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 2]
```diff
@@ -153,6 +153,10 @@ users:
     client-key: fake-key-file
 - name: experimenter
   user:
+    # Storing password in Kubernetes client config is risky.
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:156]:
```suggestion
    # Documentation note (this comment is NOT part of the command output).
    # Storing password in Kubernetes client config is risky.
```

**Review by sftim — COMMENTED** [review]
Thanks.

We should *also* warn about the insecure username and password on or about line 93. Use a [caution](https://kubernetes.io/docs/contribute/style/style-guide/#caution) callout.

**Comment by sreeram-venkitesh** [comment]
@sftim I've made the changes from the reviews and have squashed my commits. Please let me know if the caution callout is added with the proper wording and in the correct location.

File: content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 3]
```diff
@@ -88,6 +88,10 @@ kubectl config --kubeconfig=config-demo set-cluster scratch --server=https://5.6
 
 Add user details to your configuration file:
 
+{{< caution >}}
+Storing passwords in Kubernetes client config is risky. A better alternative would be to use a credential plugin and store them separately. See: [client-go credential plugins](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins)
```
Comment (Babapool) [inline content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:92]: Just a small nit from my side:
```suggestion
Storing passwords in Kubernetes client config is risky. A better alternative would be to use a credential plugin and store them separately. See: [client-go credential plugins](/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins)
```

**Comment by sreeram-venkitesh** [comment]
I've made all the changes from the reviews

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

With another LGTM from a different reviewer, I can also approve this change (which goes live as soon as we merge it).

**Comment by reylejano** [comment]
Preview:
https://deploy-preview-37339--kubernetes-io-main-staging.netlify.app/docs/tasks/access-application-cluster/configure-access-multiple-clusters/

Thank you for addressing comments/feedback

/approve

**Comment by sreeram-venkitesh** [comment]
@sftim @reylejano @Babapool Thank you! 🎉

**Pull request merged**